api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -201,6 +223,7 @@ public void updateProfileForUser(DbUser user, Profile updatedProfile, Profile pr
 
     Timestamp now = new Timestamp(clock.instant().toEpochMilli());
 
+    user.setContactEmail(updatedProfile.getContactEmail());
```

Comment:
admins can change this, but users cannot.  See the validateProfile() methods family.

Reply:
The annotation is Controller-only, but I added a UserService.hasAuthority() here which does the job